project-chip/connectedhomeip
# Access control device type support

#### Problem
Fixes #14431

#### Change overview
Add device type resolver which can be registered with access control
for looking up device types on endpoints. Provide real and test
implementations.

#### Testing
* Unit tests updated.
* Build and run chip-all-clusters-app using chip-tool on Linux
* Install ACLs for device types 0x16 (root) and 0x100 (on/off light device)
* Check that device type targets work as expected
